ProfileView/src/index.ts

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+// This file imports all of the webcomponents from "components" folder
+
+import {
+  html,
+  internalProperty,
+  property,
+  customElement,
+  LitElement
+} from "lit-element";
+// import "./components/ActivityStream/ActivityItem";
+// import "./components/View/View";
+import "@cjaas/common-components";
```

Comment:
can you import a specific component instead?

Reply:
at the moment, no, we'll need to make changes to common-component repo